src/main/java/xyz/aadev/generitech/client/gui/machines/GuiPulverizer.java

```diff
@@ -57,7 +57,6 @@
     ITeslaHolder container;
     Rectangle powerBar;
     private TileEntityPulverizer tileEntity;
-    private GuiHelper guiHelper = new GuiHelper();
     private MachineTier machineTier;
     private HashMap<Rectangle, List<String>> tooltips = new HashMap<Rectangle, List<String>>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS2293)